arrows/vxl/high_pass_filter.cxx

```diff
@@ -0,0 +1,478 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+#include "high_pass_filter.h"
+
+#include <arrows/vxl/image_container.h>
+
+#include <vil/vil_image_view.h>
+#include <vil/vil_math.h>
+#include <vil/vil_plane.h>
+#include <vil/vil_transpose.h>
+
+namespace kwiver {
+
+namespace arrows {
+
+namespace vxl {
+
```

Comment:
This is purely optional, so feel free to ignore, but there is support for 1 of n options, such as the "mode" parameter.

Need to include "enum_converter.h"
```suggestion
class enum filter_mode {
    MODE_box,
    MODE_bidir };
    
    ENUM_CONVERTER( mode_converter, filter_mode,
    { "box", MODE_box },
    { "bidir", MODE_bidir } )
```